.bcr/web_testing_go/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+bcr_test_module:
+  module_path: "../web_testing_go"
+  tasks:
+    # Only run web_testing_go tests on linux but not macos and windows
+    web_testing_go_ubuntu2004:
+      name: "rules_web_testing_go in ubuntu"
+      platform: ubuntu2004
+      shell_commands:
+      - "sh ../.bazelci/setup_ubuntu.sh"
```

Comment:
I don't think this'll work. When you check out the source of `rules_web_testing_go`, you only have the subdirectory `web_testing_go` available. In fact I don't think _any_ Bazel command here can work, because of the `local_path_override` in `web_testing_go/MODULE.bazel`. Note that `bcr_test_module` is pretending that you're fetching the module source (in this case, the `web_testing_go` subdirectory of this git repo), `cd`-ing into the `module_path` directory, and then running Bazel from that location. That means your `local_path_override` pointing to `..` would just point to a bogus location.

Reply:
IIUC, we need to alias setup_ubuntu.sh in submodules to make it available.

Reply:
I don't think symlinks would work here. You'd need a copy of the files. Symlinks, when unzipped on Linux, remain symlinks; if you're symlinking to something in `..`, and that "something" is not there, you'd still get an error.

The alternative is simply to not run the tests as part of BCR presubmit.